airflow/providers/google/ADDITIONAL_INFO.md

```diff
@@ -0,0 +1,26 @@
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+ -->
+
+## Additional limitations
+
+The [2020.11.13 release](#release-20201113) of this provider is only usable with Apache Airflow >= 1.10.12 version due to refactorings implemented in
```

Comment:
One comment here @leahecole -> I think it's not precise.

The package, in general, should be usable and installable with 1.10.12, but only the GKEPodOperaotr has some problems with <1.10.12 because cncf.kubernetes cannot be installed there (cncf.kubernetes backport provider is an optional cross-dependency for the google backport provider). So we have actually not limited the google packae to >=1.10.12 - you can still install it with airflow 1.10.9 and it will continue to work (except the GKEPodOperator)

Reply:
At least this is what we *think* it is. As far as I remember, you mentioned there are problems with *installing* the provider in composer environment with 1.10.9 so I am wondering where it came from in this case.

Reply:
I had problems installing it in a Composer environment with 1.10.10, (1.10.11 isn't offered in Composer) but not 1.10.12, @mik-laj mentioned that the latest version of this package was only compatible with the latest version of Composer but earlier version of the package are compatible with older versions of Airflow - I'm happy to have you adjust the wording as you see fit - I'm just trying to emulate the language used with the kubernetes package and the info given to me by Kamil and get it documented centrally so no one else has to figure this out the hard way 😄